We build our aarch64 package on aarch64, so we don't need to use a
cross-compiler for linking.

diff --git a/.cargo/config.toml b/.cargo/config.toml
index 803f62499..1c04c0034 100644
--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -4,8 +4,6 @@

 [target.x86_64-unknown-redox]
 linker = "x86_64-unknown-redox-gcc"
-[target.aarch64-unknown-linux-gnu]
-linker = "aarch64-linux-gnu-gcc"
 [target.riscv64gc-unknown-linux-musl]
 rustflags = ["-C", "target-feature=+crt-static"]
